### ISSUE TITLE
Add a config file

### ISSUE THREAD

**Issue by marshevms** [marshevms/mlfc#13]
Add a configuration file to restore user configuration when the program starts or after restoring the laptop from sleep

**Comment by BobbyWibowo** [comment]
Is the fact that this hasn't been implemented yet, the reason why Advanced configs are reset after a restart?

**Comment by marshevms** [comment]
Oh. Wow. Not expected. Yes it is. On New Year's holidays, I just wanted to do this and bring the program to some kind of more or less presentable form.

**Comment by BobbyWibowo** [comment]
I suppose I'll settle with Auto profile for now, at least it appeared to be working better than whatever built-in solutions Ubuntu had. And for that, thanks by the way!

All these times it didn't feel like my fans ever sped up when they had to. Fortunately, this laptop had the Cooler Booster button that would work even on Linux, so I never bothered looking further into the fans thing.
And when I finally bothered, I found out `fancontrol` package wouldn't even work on this laptop, lmao. Then I happened to stumble upon this repo as I tried to look for other alternatives, and I'm glad to say that it worked.
